README.md

```diff
@@ -1,4 +1,4 @@
-# Nix Installer
+# Determinate Nix Installer
```

Comment:
Could I cajole you into prepending `The` here as well? :)

Reply:
... Yes.